frontend/src/pages/AboutPage/AboutPage.css

```diff
@@ -0,0 +1,4 @@
+.paragraph-like {
+    font-size: 16px;
+    font-weight: 400;
```

Comment:
The default font weight in CSS is typically 400, so specifying it explicitly may be unnecessary. We can likely streamline the code by omitting it.

Reply:
Inline with accessibility criteria, I used headings to define blocks of text therefore if I don't specify the font-weight the text will be bold since I am not using <p> paragraphs.